scylladb/sphinx-scylladb-theme
# Change sidebar color to gray

## Context

> Left sidebar color is too strong

This PR adds a background gradient to the sidebar.

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser, and enter an internal page. The sidebar should look as follows:

![image](https://user-images.githubusercontent.com/9107969/141092537-bbc15c35-b2a7-4863-b78d-9222d98c815a.png)

With scrollbar:

![image](https://user-images.githubusercontent.com/9107969/141092588-5ae24b29-c48f-4523-82b7-f39331546fc5.png)

With multiversion:

![image](https://user-images.githubusercontent.com/9107969/141094070-bddcf569-4bee-425e-b653-49740da9e293.png)
